Production_scripts/Process_extracts/Produce_extract_tests/extract_chi_ltc_tests.R

```diff
@@ -0,0 +1,93 @@
+####################################################
+# Name of file - extract_chi_ltc_tests.R
+# Original Authors - Bateman McBride
+# Original Date - August 2022
+# Written/run on - RStudio Server
+# Version of R - 3.6.1
+# Description - Produce tests for source linkage files:
+#               LTC file received from IT.
+#####################################################
+
```

Comment:
```suggestion
# Load packages 

library(dplyr)
```
Because you have used dplyr a lot here, i think we can load this at the beginning to trim down the code

Reply:
I don't think we need to load `dplyr` as we're using `dplyr::` throughout.

Reply:
I think with the changes James suggested, there's a lot fewer lines starting `dplyr::` so it's not as messy